pkg/tnf/handlers/cnffsdiff/cnffsdiff.go

```diff
@@ -73,14 +78,22 @@ func (p *CnfFsDiff) ReelFirst() *reel.Step {
 func (p *CnfFsDiff) ReelMatch(pattern, before, match string) *reel.Step {
 	p.result = tnf.SUCCESS
 	switch pattern {
-	case lib:
```

Comment:
we do not have a case for "lib"? I see the regex  defined above

Reply:
fixed